index.html

```diff
@@ -9,31 +9,51 @@
     <script src="https://kit.fontawesome.com/091ffb7972.js" crossorigin="anonymous"></script>
 </head>
 <body>
+  <div class="background-desktop">
   <header class="head">
-    <a href="#" class="pagename" >Welcome</a>
-    <nav>
+    <nav class="nav-bar">
+      <div class="mobile-navhead">
+      <a href="#" class="pagename" >Welcome</a>
       <img src="tools/Union.svg" alt="" class="nav-icon">
+    </div>
+      <ul class="desktop-nav-container">
+        <li><a href="#">Portfolio</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Contact</a></li>
+      </ul>
     </nav>
   </header>
   <!-- welcome page -->
     <section class="welcome">
         <h1><span class="greetings">Hey there. I’m Josue</span><br>
         <span class="profession">I’m a software developer</span></h1>
-        <p class="description">I'm a student who is in his first week of the microverse module,
-             this is my first project I'm working on to see my advencements or progress.</p>
+        <p class="description"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+          don’t hestiate to contact me.</p>
              <ul class="socials">
               <li><a href="#"><img src="tools/twitter.svg" alt=""></a></li>
               <li><a href="#"><img src="tools/Linkedin icon.svg" alt=""></a></li>
               <li><a href="#"><img src="tools/medium.svg" alt=""></a></li>
               <li><a href="#"><img src="tools/github.svg" alt=""></a></li>
               <li><a href="#"><img src="tools/angellist.svg" alt=""></a></li>
             </ul>
-           <img src="tools/Disabled.svg" alt="">
+            <div class="dropdown">
+           <img src="tools/Disabled.svg" alt=""class="">
+          </div>
     </section>
+  </div>
+  
+       <div class="rece-indi">
+      <h2 class="recents-page">My recents works</h2>
+      <img src="tools/Indicator.png" alt="" class="indicator">
+      </div>
     <section class="mywork">
       <!-- mywork section page -->
-      <h2 class="recents-page">My recents <br>works</h2>
+      <!-- <div class="rece-indi">
+      <h2 class="recents-page">My recents works</h2>
       <img src="tools/Indicator.png" alt="" class="indicator">
+      </div> -->
+      <!-- <div class="mywork-container"> -->
```

Comment:
- [ ] Your code is organized well, but remember we never should leave a commented code on a code base. Doing so will make your code messy. Please remove these commented codes. Remember we can always go back with a commit history to check any code we have once written.